assets/js/main.js

```diff
@@ -9,13 +9,13 @@ $('body').on({
     if (submenu[0] && submenu[0].children.length) {
       $('.header-main').addClass('header-open');
     } else {
-      $('.sub-menu.open').hide();
+      $('.sub-menu.open').addClass('hidden');
     }
   },
   mouseleave: function() {
     if ($('.header-main .submenu-item > a.active').length > 0){
       $('.header-main').addClass('header-open');
-      $('.sub-menu.open').show(150);
+      $('.sub-menu.open').removeClass('hidden');
```

Comment:
150 is not necessary anymore? is there a css animation now?

Reply:
I will add something here yes.

Reply:
I've tested this and don't see the white block we've seen previously. Seems that `removeClass` takes longer than `show`

Reply:
It's horrible as it was before, I think the way of fixing this is with animations. But this seems to be outside scope just yet.

Reply:
Makes sense, thanks for the update.